Student/Student/StudentAppDelegate.swift

```diff
@@ -324,6 +331,7 @@ extension StudentAppDelegate {
 extension StudentAppDelegate: LoginDelegate, NativeLoginManagerDelegate {
     func changeUser() {
         environment.k5.userDidLogout()
+        self.environment.k5.sessionDefaults?.isK5StudentView = false
```

Comment:
I'm thinking about if we need these setter lines at all. `SessionDefaults` is a per user storage so it won't affect other users and `isK5StudentView` is updated both in `didFinishLaunchingWithOptions` and `open url` so if that test user changes to non-K5 it will be updated.

Reply:
We need this, I moved it to k5 user logout.